upup/pkg/fi/cloudup/gcetasks/instancetemplate.go

```diff
@@ -235,7 +245,7 @@ func (e *InstanceTemplate) mapToGCE(project string, region string) (*compute.Ins
 	// TODO: This is similar to Instance...
 	var scheduling *compute.Scheduling
 
-	if fi.BoolValue(e.Preemptible) {
+	if fi.BoolValue(e.Preemptible) || len(e.GuestAccelerators) > 0 {
```

Comment:
This has nothing to do with preemption. Instead, add this after the current block:
```go
if len(e.GuestAccelerators) > 0 {
    // Instances with accelerators cannot be migrated.
    scheduling.OnHostMaintenance = "TERMINATE"
}